tardis/analysis_tools/opacities.py

```diff
@@ -36,7 +36,7 @@ class opacity_calculator:
 
     Parameters
     ----------
-    mdl : tardis.model.SimulationState
+    mdl : tardis.model.Radial1DModel
```

Comment:
Unless this code is intended to only work with a legacy version of TARDIS, this should be the SimulationState object

Reply:
@andrewfullard I think this will be part of the opacities module. Maybe merge for now and then remove once that functionality is included